style/js/shield_defs.js

```diff
@@ -204,6 +217,14 @@ export function loadShields(shieldImages) {
     },
   };
 
+  shields["US:GA:Truck"] = banneredShield(shields["US:GA"], ["TRK"]);
```

Comment:
There is something not quite right with the sole truck route in GA:
![image](https://user-images.githubusercontent.com/3254090/154405600-0b18d647-b059-4749-b250-266eeff13887.png)
![image](https://user-images.githubusercontent.com/3254090/154405642-8091cd7b-6eab-4790-8060-d6549820f78c.png)